### PR TITLE
Capstone Project-Module 1

### PR DESCRIPTION
**Pull request by V0pe**
### Fashion Show Conference Website 

In this project, I did the followings,

- Add the mobile home page
- Add the mobile about page. 
- Add the desktop home and about pages.
- Implement the mobile menu icon that display the menu header.
- Add the speakers section dynamically with javascript. 
- Add some UX functionality that helps user easy access. 

The video to the website is attached to the REadme File. 

Attached is the snapshot of the website. 
image

image

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My page</a></li>
+                <li><a href="#">Logout</a></li>
+                
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-menu">
+                <a href="/index.html">
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] Currently, this link takes to a 404 page, please fix this 👍
- [ ] Also, in the mobile menu, it doesn't seem to be working at all, please fix this issue in order for the user to navigate easily between the website pages. 🙏

File: about.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                love to make creative ideas recongnized.
+                <span>We invite researchers from academy and industry, and 
+                    professionals from the world of Fashion to take part in <br>
+                    FS2022 reflecting on how to make sustainable fashion 
+                    not just a dream, but indeed a reality we are all engaged
+                     with.</span>
+            </p>
+        </section>
+        <section class="event">
+            <h1>.</h1>
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] For semantic HTML best practices, please make sure that each HTML document has only one `<h1>` tag. This also helps with SEO 👍

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
-# Conference-Capstone-
\ No newline at end of file
+# Microverse Capstone Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Fashion Show Conference website
+
+> This project describes the use of HTML, CSS and Javascript. 
+
```
Comment (MahmoudBakr23) [inline README.md:8]: ### OPTIONAL
- You can add an image or two of your app to the README file.
- Also, please fix the MIT link for a completely professional repo.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @V0pe 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linters are correct
- Great design
- Well done working with CSS media queries
- Semantic HTML
- Nice job implementing the JavaScript part of this project

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 2 above)
Comment (V0pe) [reply about.html]: - [ ] @MahmoudBakr23  Thank you for the observation. I have changed it to `<h5>` tag.

File: about.html (hunk 1 above)
Comment (V0pe) [reply about.html]: - [ ] @MahmoudBakr23 Thanks for the observation. I have rectified it. Apparently, I didn't specify the directory well.

**Review by zainsadaqat — APPROVED** [review]
# Hello @V0pe  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**